README.md

```diff
@@ -24,10 +24,99 @@ An example manuscript with documentation comments is below:
+2) The layout/design of the catalog is subject to change, which may show the thumbnails at larger sizes.
+3) In general, we want to future-proof the catalog against increasing image resolution standards, and for Manubot's own changing needs.
+
+- **PNG format**
+Provide your image as a `.png` file.
+This avoids unsightly artifacts produced by other common image formats while still keeping file size to a minimum.
+
+- **Select a striking image**
+Select a portion of your manuscript that is interesting and unique.
+Your thumbnail is not expected to accurately capture or explain what your manuscript is about.
+The purpose of the thumbnail is to visually distinguish your paper from other papers, so that readers can easily remember and quickly identify it among many others.
+As such, the first preference is for a unique, colorful figure, without its caption.
+If your manuscript has no figures, use an interesting-looking table, code block, or set of equations.
+If your manuscript has none of those, use any other section that looks unique in some way; more than just a paragraph of black text.
+If your manuscript is all just plain text, don't provide a thumbnail at all; a placeholder thumbnail will be used automatically.
+
+- **Don't stretch**
+Don't distort the original aspect ratio of your image; not even a little.
+It looks bad and is very noticeable.
+Asymmetrical horizontal and vertical whitespace is much preferable to a stretched image.
+If you are struggling to meet the dimension requirement, refer to the guidelines on cropping.
+
+- **Don't scale**
+You should always create, save, and maintain your figures in vector format where possible.
+This allows the image to be scaled to any size without loss of quality.
+If your thumbnail source is a vector image, scale it to the required dimensions *before* converting/rendering it to `.png`.
+Do not, for example, render your vector image to a 800x800 `.png` and then scale it up to 1000x1000.
+If your thumbnail has to be a raster image, it still must meet the dimension requirements above, and may be rejected by the catalog maintainers if it is noticeably up-scaled.
+
+- **Don't include paragraph text or captions**
+Text, especially plain black text, serves little to no purpose in a thumbnail.
+Thumbnails are meant to be quick visual identifiers, not snippets of detailed content.
+Don't include text in your thumbnail, unless it is baked into the figure itself (not its caption), or unless it is necessary per the "striking image" guideline above.
+
+- **Frame/crop nicely**
+Cropping an image nicely is more of an artform than you might think.
+As such, it's hard to specify a set of hard/quantifiable rules about what looks good.
+If you're familiar with [frame composition](https://en.wikipedia.org/wiki/Composition_(visual_arts)) from art/photography/cinematography, the same techniques can be applied here.
+In addition, here are some general guidelines to follow:
+
+  - **Choose a good aspect ratio**
```

Comment:
Is the indentation on these bullets necessary? These are sub-bullets perhaps?

Reply:
Yes, those are sub-guidelines of the main "crop things nicely" guideline.